getting_started/audio.rst

```diff
@@ -14,45 +15,46 @@ All of this is supported by Escoria.
 Playing music and sound effects
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Escoria features three built-in audio players, which are accessible by their
+Escoria features three built-in audio players which are accessible by their
 ESC global IDs:
 
-- _music: Used for looping background music
-- _sound: Used for playing sound files
-- _speech: Used for voice support
+- `_music`: Used for looping background music
```

Comment:
Please use `` here as that's the ReST standard.

Reply:
I actually knew that. That's what I get for working on this just before sleeping.